Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -85,7 +88,7 @@ public struct OnboardingView<TitleView: View, ContentView: View, ActionView: Vie
     }
     
     /// Creates the default style of the `OnboardingView` uses a combination of an ``OnboardingTitleView``, ``OnboardingInformationView``,
-    /// and ``OnboardingActionsView``.
+    /// and ``OnboardingActionsView``. This intitializer uses localized stirings for the title, subtitle, and actionText.
```

Comment:
We generally don't refer to the agreements in the main body of the fiction, they are referred to in the main part of the parameters section.